snippets/distr-tracing-tempo-secret-example.adoc

```diff
@@ -0,0 +1,23 @@
+// Text snippet included in the following modules:
+//
+// * distr-tracing-tempo-install-web-console.adoc
+// * distr-tracing-tempo-install-cli.adoc
+
+:_content-type: SNIPPET
+
+.Secret for Amazon S3 and MinIO storage
+====
```

Comment:
This is a standard AsciiDoc formatting for examples.
If you have `.Secret for Amazon S3 and MinIO storage ====`, the heading will be rendered as `Example 5. Secret for Amazon S3 and MinIO storage".
If you have `.Secret for Amazon S3 and MinIO storage ----`, the heading will be rendered as `Secret for Amazon S3 and MinIO storage".
Anyway, I've removed the `====` syntax from all of my examples here that don't have introductory text.

Reply:
Thanks! I talked to a team member with more history on OCP, and he says the team used this convention in version 3 docs, but have moved away from it since then and would discourage bringing it back :slightly_smiling_face: